cv_next/app/cv/[cvId]/components/commentSection/commentForm.tsx

```diff
@@ -0,0 +1,57 @@
+"use client"
+import { addNewComment } from "@/app/actions/comments/addNewComment"
+import { useSupabase } from "@/hooks/supabase"
+import { useRef } from "react"
+import { RxPaperPlane } from "react-icons/rx"
+import { mutate } from "swr"
+
+export default function CommentForm({ cv }: { cv: CvModel }) {
+  const formRef = useRef<HTMLFormElement | null>(null)
+  const supabase = useSupabase()
+
+  const formAction = async (formData: FormData) => {
+    const userId = (await supabase.auth.getUser()).data.user?.id
+    if (!userId) throw new Error("User not found") // TODO: handle this
```

Comment:
Do not forget to handle this before merge (e.g. log error instead of throw error)

Reply:
As far as I know this was done in the comments branch that improved several things here
I will make sure tho(or a least add it as a thing to handle there)